src/app/data-planes/components/DataPlaneList.vue

```diff
@@ -77,28 +81,34 @@
       </template>
     </template>
 
-    <template #status="{ row }: { row: DataPlaneOverviewTableRow }">
+    <template #certificate="{ row }: { row: DataplaneOverview }">
+      <template v-if="row.dataplaneInsight.mTLS?.certificateExpirationTime">
```

Comment:
**Comment**: I’m considering to always set `mTLS` o we can save the optional chaining for stuff like this.

Reply:
Q: What's your thinking on how we would have things without mTLS set?

Reply:
I'm guessing that mTLS can sometimes be empty/absent to mean that mTLS isn't enabled on this Dataplane?

Reply:
Yes, that is the case. That’s one of the reasons why I didn’t change this in this PR. I might not change it at all if we rely on this logic. An alternative would be:

- introduce a field `hasMtlsEnabled: boolean` to `DataplaneOverview` and use that for such checks
- always write a (possibly empty) `mTLS` object onto `dataplaneInsight` so that accesses like this one are simpler (i.e. don’t need to use optional chaining)

In any case, that’s definitely something that I want to consider separately from this PR.

Reply:
Cool, gotcha, ok yeah defo something worth considering, thanks for sharing your thinking